vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -566,6 +568,11 @@ public Registration addDetachListener(
         return super.addDetachListener(listener);
     }
 
+    private void setDimension(String dimension, String value) {
+        getElement()
+            .executeJs("this.$.overlay.$.overlay.style."+dimension+"=$0", value);
```

Comment:
This could be aligned as: `getElement().executeJs("this.$.overlay.$.overlay.style[$0]=$1", dimension, value);`

Reply:
Changed as suggested.